docs/content/misc/math-expr.md

```diff
@@ -85,6 +85,8 @@ The following built-in functions are available.
 |upper|upper(expr) converts a string to uppercase|
 |reverse|reverse(expr) reverses a string|
 |repeat|repeat(expr, N) repeats a string N times|
+|lpad|lpad(expr, length, chars) returns a string, left-padded to a specified length with the specified characters; or, when the string to be padded is longer than the length specified after padding, only that portion of the expression that fits into the specified length.
```

Comment:
The documentation for these functions appears to be a near word for word copy of the Oracle documentation of these functions, with 'expression' replaced with 'string' in most cases. I think this is risky, https://www.oracle.com/legal/copyright.html indicates that both code and documentation is covered, and I think these descriptions need rewritten.

Maybe something like 
>lpad(expr, length, chars) returns a string of length from expr left-padded with chars. If either expr or chars are null, the result will be null.

Reply:
Thanks a lot for reminding, @clintropolis . I totally agree with you and updated the doc.